static/docs/use-cases/data-registries.md

```diff
@@ -0,0 +1,195 @@
+the same level as with source code (commit history, branching, pull requests,
+reviews, etc.):
+
+```dvc
+$ git add music/songs.dvc music/.gitignore
+$ git commit -m "Track 1.8 GB 10,000 song dataset in music/"
+```
+
+> The actual data is stored in the project's <abbr>cache</abbr> and should be
+> [pushed](/doc/command-reference/push) to one or more
+> [remote storage](/doc/command-reference/remote) locations.
+
+## Using registries
+
+The main methods to consume <abbr>data artifacts</abbr> from a **data registry**
+are the `dvc import` and `dvc get` commands, as well as the `dvc.api.open()`
+function (Python).
+
+### Simple download (get)
+
+This is analogous to using direct download tools like
+[`wget`](https://www.gnu.org/software/wget/) (HTTP),
+[`aws s3 cp`](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html) (S3),
+etc. To get a dataset for example, we can run something like:
+
+```dvc
+$ dvc get git@git-server.url:path/to/repository.git \
+          path/to/dataset
+```
+
+This downloads `path/to/dataset` from the <abbr>project</abbr>'s
+[default remote](/doc/command-reference/remote/default) and places it in the
+current working directory (anywhere in the file system with user write access).
+
+> Note that this command (as well as `dvc import`) has a `--rev` option to
+> download specific versions of the data.
+
+### Import workflow
+
+`dvc import` uses the same syntax as `dvc get`:
+
+```dvc
+$ dvc import git@git-server.url:path/to/repository.git \
+             path/to/dataset
+```
+
+> Note that unlike `dvc get`, which can be used from any directory, `dvc import`
+> needs to run within an [initialized](/doc/command-reference/init) DVC project.
+
+Besides downloading, importing saves the dependency of the local project towards
+the data source (registry repository). This is achieved by creating a particular
+kind of [DVC-file](/doc/user-guide/dvc-file-format) (a.k.a. _import stage_).
+This file can be used staged and committed with Git.
+
+As an addition to the import workflow, and enabled the saved dependency, we can
+easily bring it up to date in our consumer project with `dvc update` whenever
+the the dataset changes in the source project (data registry):
+
+```dvc
+$ dvc update dataset.dvc
+```
+
+`dvc update` downloads new and changed files, or removes deleted ones, from
+`path/to/dataset` based on the latest version of the source project. It also
+updates the project dependency metadata in the import stage (DVC-file).
+
+### Programatic reusability of DVC data
+
+Our Python API, included with the `dvc` package installed with DVC, includes the
+`open` function to load/stream data directly from remote DVC projects:
+
+```python
+import dvc.api.open
+
+model_path = 'path/to/model'
+repo_url = 'git@git-server.url:path/to/repository.git'
+
+with dvc.api.open(model_path, repo_url) as model:
+    # Make some predictions...
+```
+
+This opens `path/to/dataset` as a file descriptor. Such a method could be used
```

Comment:
`path/to/model`? .. better use `model.pkl` ... if we use our example-get-started we can even provide a real example

Reply:
Changing to `nlp/model.pkl` (in hypothetical `https://github.com/iterative/model-registry` repo).

Avoiding real examples so we don't have to make sure the commands are reproducible.

Reply:
Changing to `https://github.com/example/registry` per Slack conversation. Applying similar changes in the other URLS.